src/App.tsx

```diff
@@ -16,35 +18,77 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section content">
-      <div className="buttons">
-        <button type="button" className="button is-info is-light">
-          Sort alphabetically
-        </button>
+  const [sortField, setSortField] = useState('');
+  const [isReversed, setReverse] = useState(false);
+  const [isReset, setReset] = useState(goodsFromServer);
```

Comment:
```suggestion
  const [initialGoods, setInitialGoods] = useState(goodsFromServer);
```
Consider renaming the state (isSomething, hasSomething - should be boolean states)